<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NAT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Src address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Chain</string>
    </property>
   </widget>
   <widget class="QListWidget" name="actionField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="srcaddrField">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>101</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QListWidget" name="protField">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>81</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="chainField">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>111</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="dstaddrField">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>101</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>60</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>90</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="srcacceptButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>0</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add srcnat accept rule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dstacceptButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>30</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add dst-nat accept rule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dstButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>60</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add dst-nat </string>
    </property>
   </widget>
   <widget class="QPushButton" name="srcButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>90</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add src-nat </string>
    </property>
   </widget>
   <widget class="QPushButton" name="masqButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>120</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add masquarade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dst address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Protocol</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Src port</string>
    </property>
   </widget>
   <widget class="QListWidget" name="srcportField">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>81</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dst port</string>
    </property>
   </widget>
   <widget class="QListWidget" name="dstportField">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>81</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>To address</string>
    </property>
   </widget>
   <widget class="QListWidget" name="toaddressField">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>81</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_8">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>To ports</string>
    </property>
   </widget>
   <widget class="QListWidget" name="toportField">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>81</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_9">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dynamic</string>
    </property>
   </widget>
   <widget class="QListWidget" name="dynamicField">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>81</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
